Sources/Services/ContainerAPIService/Containers/ContainersService.swift

```diff
@@ -512,6 +512,36 @@ public actor ContainersService {
         }
     }
 
+    public func prune() async throws -> ([String], UInt64) {
+        self.log.debug("\(#function)")
+        let containers = self.containers
```

Comment:
nit: This can be combined into one line

```suggestion
        let stoppedContainers = self.containers.filter { $0.value.snapshot.status == .stopped }
```